v2/bigtable-changestreams-to-hbase/README.md

```diff
@@ -8,17 +8,33 @@ The [BigtableToHBase](src/main/java/com/google/cloud/teleport/v2/templates/Bigta
 * Bigtable table with change streams enabled
 * Hbase instance that is accessible from Dataflow
 
-#### Beware
+#### Beware - provision Hbase resources to handle Dataflow traffic
 
 * Dataflow when strongly provisioned can translate Bigtable input QPS directly to Hbase, the user should configure Hbase so that Hbase can handle that QPS without crashing.
+
 * Bigtable change streams can handle very large single mutations. Hbase write buffers should be adequately provisioned so as not to cause exceptions when writing in large mutations.
 
+#### Beware - out-of-order writes can occur in some scenarios
+
+The replicator writes to Hbase as a side effect in Dataflow. If Dataflow loses connection to a worker, another worker will be spun up and a change stream mutation could be duplicated. In the following scenario, a duplication could occur:
+  * Dataflow spins up worker A
+  * Worker A replicates a Put
+  * Dataflow loses connection to worker A before the Put is replicated and spins up worker B
+  * Dataflow assigns the Put to worker B, now 2 workers write the same Put
+
+The above scenario is not a problem most times. However, if a major compaction occurs and a cell had a masking Delete, then a delayed Put will be applied to a cell that should've been deleted. This scenario is theoretically possible but extremely unlikely.
```

Comment:
I think we should make it more clear that there are multiple scenarios here, the other important one being if you have two puts with the same timestamp and different values out of order

Reply:
@jackdingilian 2 mutations with same timestamp is a problem with HBase <-> HBase replication as well. See https://issues.apache.org/jira/browse/HBASE-9465

Reply:
the only realistic problem I see is "put (ts=0), delete (ts=1), major compaction, put(ts=0)" creates a zombie put.